hartshorn-core/src/main/java/org/dockbox/hartshorn/core/context/element/TypeContext.java

```diff
@@ -380,6 +380,16 @@ public boolean isNative() {
         return this.isNative;
     }
 
+    public boolean isArray() {
+        return this.type().isArray();
```

Comment:
This can be moved to the constructor, assign to a field annotated with `@Getter` like we did for enums, annotations etc
https://github.com/GuusLieben/Hartshorn/blob/a48c2eafe4240e3bd91713ca952b1ae9bc95defc/hartshorn-core/src/main/java/org/dockbox/hartshorn/core/context/element/TypeContext.java#L133